2-mandatory/6-budgets.js

```diff
@@ -11,6 +11,8 @@ Should give return the answer of 62600.
 **/
 
 function getBudgets(peopleArray) {
+  let sumBudget = peopleArray.map(a => a.budget).reduce((x,y) => x+y,0);
```

Comment:
Well done for using reduce! Probably don't need the map too. No need to assign to a varaible. Simply return the result. And try to use meaningful varaible names e.g.

```
return peopleArray.reduce((total, person) => total + person.budget, 0);
```

Reply:
thank you for your feedback. It is better than mine.